css/amp-story-player.css

```diff
@@ -0,0 +1,37 @@
+amp-story-player {
+  background: var(--story-player-poster, lightgrey);
+  width: 405px;
+  height: 720px;
```

Comment:
We'll have to discuss the actual default size we want: on a regular iPhone, this will be bigger than the viewport.

Reply:
/cc @hongwei1990